plugin/trino-iceberg/pom.xml

```diff
@@ -401,6 +429,8 @@
                         <!-- com.amazonaws:aws-java-sdk-core and software.amazon.awssdk:sdk-core MIME type file duplicate-->
                         <ignoredResourcePattern>mime.types</ignoredResourcePattern>
                         <ignoredResourcePattern>about.html</ignoredResourcePattern>
+                        <ignoredResourcePattern>iceberg-build.properties</ignoredResourcePattern>
+                        <ignoredResourcePattern>mozilla/public-suffix-list.txt</ignoredResourcePattern>
```

Comment:
Can we file an issue for this in Iceberg? This resource should be namespaced to avoid conflicts, or drop it entirely (since Iceberg doesn't use the public suffix functions).

Reply:
I'll bring this up because the `iceberg-build.properties` was recently introduced and is included in all the artifact jars so that it's clear where any given jar came from.

The suffix list is a conflict from http client libraries, I believe.

Reply:
Ah, I was thinking of the Guava `com/google/thirdparty/publicsuffix` package which is easy to miss since it's different than `com/google/common`. If Iceberg isn't the one shading the HTTP client, then there's nothing we can fix here.

For the `iceberg-build.properties`, you might consider putting it in `META-INF`, similar to how Maven does it with e.g., `META-INF/maven/com.google.guava/guava/pom.properties`.